Makefile

```diff
@@ -112,6 +114,16 @@ tarball: npm_licenses common-tarball
 .PHONY: docker
 docker: npm_licenses common-docker
 
+.PHONY: local-docker-image $(BUILD_LOCAL_DOCKER_ARCHS)
+local-docker-image: $(BUILD_LOCAL_DOCKER_ARCHS)
+$(BUILD_LOCAL_DOCKER_ARCHS): local-docker-image-%: 
+	promu crossbuild-natively common-docker-$* npm_licenses 
+
+.PHONY: crossbuild-natively
+crossbuild-natively:
```

Comment:
Hm, it's a bit inconsistent e.g. we don't have `crossbuild` Makefile cmd. Can we remove it?